src/objects/Batch_Template__c.object

```diff
@@ -213,6 +237,16 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
+        <fullName>Require_Total_Match__c</fullName>
+        <defaultValue>false</defaultValue>
+        <description>When checked, BGE grid validates the expected total against the actual total and prohibits the batch from processing if totals do not match.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>When checked, BGE grid validates the expected total against the actual total and prohibits the batch from processing if totals do not match.</inlineHelpText>
```

Comment:
@smarovich can you review this inline help text for the Require Total Match helptext field on the Batch Template page too?